src/components/PredictionGame.tsx

```
@@ -0,0 +1,187 @@
+												});
+											}}
+											disabled={isSubmitting}
+											className={cn(
+												"px-3 py-1.5 rounded-md text-xs font-medium transition-all duration-200 border",
+												isSelected
+													? "bg-purple-600 border-purple-500 text-white shadow-[0_0_10px_rgba(147,51,234,0.5)] scale-105"
+													: "bg-gray-700/50 border-gray-600 text-gray-400 hover:bg-gray-600 hover:text-gray-200 hover:border-gray-500",
+												isSubmitting && "opacity-50 cursor-not-allowed"
+											)}
+											title={rating.name}
+										>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)